### ISSUE TITLE
[BUG] Adding nested color in tailwind.config triggers `tailwindcss/no-custom-classname` warning

### ISSUE THREAD

**Issue by boomikuma** [francoismassart/eslint-plugin-tailwindcss#145]
**Describe the bug**
Adding nested color in tailwind.config triggers `tailwindcss/no-custom-classname` warning

**To Reproduce**
1. Add the following to tailwind.config file

```
colors: {
    transparent: 'transparent',
    current: 'currentColor',
    white: '#FFFFFF',
    black: '#000000',
    example: {
      1: '#F0f025',
    },
    category: {
      example: {
        1: '#F0F025',
      },
    },
}
```
2. Add `text-category-example-1` to className

**Expected behavior**
Should not trigger eslint warning

**Screenshots**
This doesn't work
<img width="812" alt="image" src="https://user-images.githubusercontent.com/37561295/172534545-f6c145bf-1e0a-499d-bb21-3218d04ed847.png">
This works
<img width="671" alt="image" src="https://user-images.githubusercontent.com/37561295/172534972-739e70ff-00e7-4342-8902-2304f31c2c36.png">

**Environment (please complete the following information):**
 - OS: macOS
 - Softwares + version used:
     -  VSCode 1.67

**Additional context**

**eslint config file or live demo**
By providing a link to a live demo, a demo video or a github repo fixing the issue will be easier.

**Comment by francoismassart** [comment]
WIP on #132

**Comment by francoismassart** [comment]
I just published a beta version with the fixes listed on #132 

`npm i eslint-plugin-tailwindcss@3.6.0-beta.0`

Please give it a try and provide feedback and or issues 👍 
Thank you

**Comment by francoismassart** [comment]
Should be fixed in the [latest release `3.6.0`](https://github.com/francoismassart/eslint-plugin-tailwindcss/releases/tag/v3.6.0)

Make sure to:

1. Update the `eslint-plugin-tailwindcss` package
2. Restart your editor
3. Report any issue on this repo

**Comment by damianobarbati** [comment]
@boompikachu how did you solve this problem? I still see this today, my custom colors trigger the error
